app/styles/ui/rebase-status.scss

```diff
@@ -0,0 +1,68 @@
+.rebase-status-component {
+  .rebase-status-icon-container {
+    height: 20px;
+    position: relative;
+    text-align: center;
+
+    .rebase-status {
+      background: var(--background-color);
+      border-width: 0 var(--spacing-half);
+      border: solid transparent;
+      box-sizing: content-box;
```

Comment:
Any particular reason why we're changing the sizing mode here? Inconsistent `box-sizing` usually leads to hard-to-debug visual problems.

Reply:
No particular reason - this came from [`app/styles/ui/_merge-status.scss`] and I'd be 👍 to potentially merging these styles into one. How urgent is this to tackle here?

Reply:
I don't have any more context on this aside from it was initially added in https://github.com/desktop/desktop/commit/d73c8a0b7fbb99508f04b04f87dad227a2be5adf - @donokuda can you recall why we settled on this?

Reply:
> How urgent is this to tackle here?

Not urgent now that I understand that this originates from `_merge-status.scss`. Let's tackle this separately.

Reply:
I've opened #7216 with the relevant bits discussed here.